.github/workflows/build.yaml

```
@@ -16,20 +16,28 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.3" ]
-                symfony: [ "^5.4.21", "^6.4" ]
-                sylius: [ "~1.13.0", "~1.14.0" ]
+                php: ["8.3"]
+                symfony: ["^5.4", "^6.4", "^7.1"]
```

```suggestion
                symfony: ["^5.4", "^6.4", "^7.2"]
```
Wouldn't it make sense to reflect this, since Symfony 7.2 has already been released?

---

Here Symfony 7.2 will be installed anyway, and this entry reflects the current dependency in composer and also the one 2.0 has, so I would leave it as it is for now and bump it up in the future